ACMAS/Docker deployment instructions.md

```diff
@@ -0,0 +1,20 @@
+# Deployment instructions for ACMAS @ RCOS v2 (Django, Docker, Postgres, Gunicorn, Nginx)
+## Production server
+_Note: Running migrations and serving static files are now handled in entrypoint.prod.sh. and entrypoint.sh._
+
+[Docker](https://docs.docker.com/get-docker/) is a pre-requisite installation to run the deployment server.
+If you do not have Docker installed, please do so before continuing to the remainder of the instructions.
+### Build images and spin up production containers
+`docker-compose -f docker-compose.prod.yml up -d --build`
+### Bring the containers down once done
```

Comment:
this needs to be more clear that this is how you end the docker program - I didn't understand that when I read it

Reply:
also need to make sure to tell people they ***have to*** remember to stop the program once they are done

Reply:
Resolved